### PR TITLE
Check in/87413/landing page fixes

### PR DESCRIPTION
**Pull request by wullaski**
## Summary

- Changes header for vista appointments
- Show subtitle only when complete
- Add eyebrow to pre check in demographics pages

## Related issue(s)

[department-of-veterans-affairs/va.gov-team#87413](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87413)
[department-of-veterans-affairs/vets-website#87410](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87410)
[department-of-veterans-affairs/va.gov-team#87412](https://github.com/department-of-veterans-affairs/va.gov-team/issues/87412)

## Testing done

- Visual
- Unit

## What areas of the site does it impact?

Checkin

## Acceptance criteria

- [ ] content matches figma files in linked issues

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/check-in/components/pages/AppointmentDetails/AppointmentDetails.unit.spec.jsx [hunk 1]
```diff
@@ -355,7 +391,7 @@ describe('check-in experience', () => {
             .exist;
         });
       });
-      describe('CVT pre-check-in appointment', () => {
+      xdescribe('CVT pre-check-in appointment', () => {
```
Comment (brianseek) [inline src/applications/check-in/components/pages/AppointmentDetails/AppointmentDetails.unit.spec.jsx:394]: what's going on with all these `xdescribe` changes? Does that do a skip?

**Review by brianseek — CHANGES_REQUESTED** [review]
I think most things look fine. I have a question about all the `xdescribe` in the tests. Also In pre-check-in I see this message about check-in on a detail page (see screenshot). That should never show in pre-check-in.
localhost_3001_health-care_appointment-pre-check-in__id=46bebc0a-b99c-464f-a5c5-560bc9eae287 (4)

File: src/applications/check-in/components/pages/AppointmentDetails/AppointmentDetails.unit.spec.jsx (hunk 1 above)
Comment (wullaski) [reply src/applications/check-in/components/pages/AppointmentDetails/AppointmentDetails.unit.spec.jsx]: Screenshot 2024-07-03 at 9 18 16 AM
😂 that's one way to fix the tests, Sometimes I do that to isolate the one test that isn't working. I forgot to go back and remove those.

**Review by brianseek — APPROVED** [review]
LGTM!

**Pull request merged**